Gemfile.lock

```diff
@@ -1,17 +1,18 @@
-GEM
-  remote: https://rubygems.org/
-  specs:
-    rake (11.2.2)
-    sass (3.4.22)
-    scss_lint (0.48.0)
-      rake (>= 0.9, < 12)
-      sass (~> 3.4.15)
-
-PLATFORMS
-  ruby
-
-DEPENDENCIES
-  scss_lint (~> 0.48.0)
-
-BUNDLED WITH
-   1.12.5
+GEM
+  remote: https://rubygems.org/
+  specs:
+    rake (11.2.2)
+    sass (3.4.22)
+    scss_lint (0.48.0)
+      rake (>= 0.9, < 12)
+      sass (~> 3.4.15)
+
+PLATFORMS
+  ruby
+  x64-mingw32
+
+DEPENDENCIES
+  scss_lint (~> 0.48.0)
+
+BUNDLED WITH
+   1.15.1

```

Comment:
Global line break change - contents are the same as previously. + Seems that after installing new Bundle (I didn't have Ruby before), it also updated the file automatically, after running the bundle.